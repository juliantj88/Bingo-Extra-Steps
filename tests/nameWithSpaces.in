1
Solomon Northup
Chris Kyle
save nameWithSpaces
4