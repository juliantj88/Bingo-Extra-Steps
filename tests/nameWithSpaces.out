Solomon Northup
Chris Kyle
0
0
1
F...............
RRRY
BBYU
BYYY
LUUU
RRLU
byrul
lbyru
ulbyr
rulby
yrulb
byrul
lbyru
ulbyr
rulby
yrulb
.
..
...
....
.....
.
..
...
....
.....
       
       

RRLBBYRLLUYBURRBLRYRBLBUUBRLBLULYRLYLLUYYULLLRLUYBLBYYYRBUURUBBYUBYRURBYYUBBURLL
0